pvlib/tests/test_losses.py

```diff
@@ -90,3 +97,101 @@ def test_soiling_hsu(rainfall_input, expected_output_2):
                          rain_accum_period=pd.Timedelta('3h'))
 
     assert_series_equal(result, expected)
+
+
+@pytest.fixture
+def greensboro_rain():
+    # get TMY3 data with rain
+    greensboro = read_tmy3(DATA_DIR / '723170TYA.CSV', coerce_year=1990)
+    # NOTE: can't use Sand Point, AK b/c Lprecipdepth is -9900, ie: missing
+    return greensboro[0].Lprecipdepth
+
+
+@pytest.fixture
+def expected_kimber_soiling_nowash():
+    return pd.read_csv(
+        DATA_DIR / 'greensboro_kimber_soil_nowash.dat',
```

Comment:
why `.dat` instead of `.csv`?

Reply:
`*.csv` is ignored by Git

https://github.com/pvlib/pvlib-python/blob/64a6498b689d27d77bb5967e13eab9bc0018d68e/.gitignore#L82

So I have to force commit csv files, but `.dat` isn't. Sorry, me lazy again. Also I kinda like dat since it kinda sounds/looks like data, so it feels like it belongs in the data folder. Silly

Reply:
I have a few reasons to prefer csv to dat... I have to guess at what I need to open a `.dat` file. If I run `head file.dat` is it going to print binary gibberish and maybe nuke my terminal session? And what is in a csv file if not data? 

We could change the ignore to be more like the notebook rule

https://github.com/pvlib/pvlib-python/blob/64a6498b689d27d77bb5967e13eab9bc0018d68e/.gitignore#L87-L89

so 

```
 # Ignore some csv 
 *.csv 
 !pvlib/data/*.csv 
```

That being said, I've previously added `.dat` files for SURFRAD and SOLRAD tests because it follows the naming convention for the raw data from those networks. Maybe those should be changed - I don't know if consistency is preferable there or not.